src/lib/components/Snackbar/index.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview A Material snackbar for showing notifications.
+ */
+
+import {html} from "lit-element";
+import {BaseElement} from "../BaseElement";
+
+const OPENING_ANIMATION_TIME = 1500;
+const CLOSING_ANIMATION_TIME = 75;
+
+class Snackbar extends BaseElement {
+  static get properties() {
+    return {
+      animatable: {type: Boolean, reflect: true},
+      stacked: {type: Boolean, reflect: true},
+      type: {type: String},
+      action: {type: Object}, // action is a Function
+    };
+  }
+
+  get open() {
+    return this.hasAttribute("open");
+  }
+
+  set open(val) {
+    let ms;
+    if (Boolean(val)) {
+      this.setAttribute("open", "");
+      ms = OPENING_ANIMATION_TIME;
+    } else {
+      this.removeAttribute("open");
+      ms = CLOSING_ANIMATION_TIME;
+    }
+
+    this.animatable = true;
```

Comment:
There's a tiny race condition here (we clear `animatable` for the first anim, not the last) if you toggle this lots but it's probably fine.

Reply:
yeah my hunch is this won't be a high frequency toggle. One thing we may look into in the future is queuing snackbars (I think devsite does this). But for now we only have 1.